app/models/bulkrax/csv_entry_decorator.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
```

Comment:
Per the playbook, it's customary to put override documentation right at the top in a comment
 like # OVERRIDE Hyrax v3.4 to add selectable themes

Reply:
https://github.com/samvera-labs/bulkrax/pull/670 was just merged, so this file shouldn't be necessary anymore. 🙂